docs/modules/reference/pages/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -124,6 +124,10 @@ In case of an HTTPS request, this is also the virtual domain to send as part of
 Colon-separated credentials such as USERNAME:PASSWORD.
 | n/a
 
+| preselect-auth
+| Preselect authentication method. At the time allows only "basic" what corresponds to Basic Authentication and requires that user name and password are provided in the attribute `user-info`
```

Comment:
```suggestion
| Preselect authentication method. 
When specified, allows only basic authentication.
User name and password must be provided in the `user-info` attribute.
```
do you mean "currently" when you say "at the time" or how I changed it. I'm also not clear if "basic" is a specific value.